homeassistant/components/sensor/nut.py

```diff
@@ -209,11 +215,12 @@ def unit_of_measurement(self):
     def device_state_attributes(self):
         """Return the sensor attributes."""
         attr = dict()
-        attr[ATTR_STATE] = self.opp_state()
+        attr[ATTR_STATE] = self.display_state() + " (" + \
+            self._data.status[KEY_STATUS] + ")"
```

Comment:
Please use `format()` instead of concats (`+`)

Reply:
Sure, makes sense.